README.md

```diff
@@ -28,24 +26,19 @@ by adding `opentelemetry_tesla` to your list of dependencies in `mix.exs`:
 ```elixir
 def deps do
   [
-    {:opentelemetry_tesla, "~> 0.1.0-rc.1"}
+    {:opentelemetry_tesla, "~> 1.1.0-rc.1"}
   ]
 end
 ```
 
 ## Setup
-In your application start:
+If you want to use Telemetry handler, make sure you add the following lines to your application start:
 ```elixir
-def start(_type, _args) do
-  OpenTelemetry.register_application_tracer(:my_telemetry_api)
-  OpentelemetryTesla.setup()
-  children = [
-    {Phoenix.PubSub, name: MyApp.PubSub},
-    MyAppWeb.Endpoint
-  ]
-  opts = [strategy: :one_for_one, name: MyApp.Supervisor]
-  Supervisor.start_link(children, opts)
-end
+OpenTelemetry.register_application_tracer(:my_telemetry_api)
+OpentelemetryTesla.setup()
 ```
 
-After this, spans will start to be created whenever a request is completed or if it eventually fails with an exception.
+On the other hand, if you want to use the middleware, just add it like any other Tesla middleware.
```

Comment:
I would say something in lines of `To also propagate the tracing information you'll also have to add the tesla middleware:`. Just to be more clear that combining these will do both the span creation and also propagation.